doc/InMemoryDbBackupAndRestore.md

```diff
@@ -0,0 +1,44 @@
+# In-memory database backup and restore of IoT Edge system modules
```

Comment:
Should we instead have one doc that talks about IoT Edge storage configuration?

Reply:
I initially had that thought but I wasn't sure what all should be documented with respect to storage configuration (in terms of details and implementation) so I went ahead with this instead.

We can add that document later and refer to it in these other docs.